docs/workflow.md

```diff
@@ -16,6 +16,14 @@ To commit the PR to the repo use the GitHub `Squash and Merge` button. We can't
 - *Do not* merge too quickly. Wait for at least 24h after the last significant changes before merging unless the change is urgent.
 - *Do* address all feedback. Not necessarily by accepting it, but by reaching a resolution with the reviewer. All comments need to be marked as resolved before merging.
 - *Do* use GitHub's tooling. Re-request review after all feedback has been addressed.
+- *Do* pay special attention to the commit message. Ensure the merge message is appropriate and helpful to the future reader. See [merge commit considerations](#merge-commit-considerations).
+
+#### Merge commit considerations
+
+GitHub merges have 2 means to specify a commit message when squash merging. Inspect both! In most scenarios, you will want to delete the commit by commit messages. Only leave the messages when they are helpful to a user in the future.
+
+![Good Commit Message](resources/good-commit-message.png)
```

Comment:
Should be delete all optional description?
Could be still have some short useful details addition to main above?

Reply:
OK, ready now. 

let me know what you think.